worker/jobs/crawling.go

```diff
@@ -0,0 +1,33 @@
+package jobs
+
+import (
+	. "google-scraper/models"
+	. "google-scraper/services/crawler"
+
+	"github.com/beego/beego/v2/core/logs"
+	"github.com/gocraft/work"
+)
+
+type Context struct{}
+
+// Number of retry
+const MaxFails = 3
+
+func (c *Context) PerformCrawling(job *work.Job) error {
```

Comment:
Can we add unit test for this function? I know that you have unit test for each function here. I think we can improve this by using `gomock` to mock the function and just test only this function in another ticket.

Reply:
Added here 6a49eb2